layouts/partials/head/twitter.html

```diff
@@ -0,0 +1,15 @@
+{{ if .Site.Params.Settings.pageTwitter}}
+<meta name="twitter:site" content="{{ .Site.Params.Settings.pageTwitter}}">
+{{ else }}
+<meta name="twitter:site" content="{{ .Site.Params.social.twitter}}">
```

Comment:
For `twitter:site` tag, we should just use `.Site.Params.opengraph.twitter.page`. Is the fallback to `.Site.Params.social.twitter` really necessary?

Reply:
I think so. If the user only has his personal account set, without the page/website account.

Reply:
Still, in that case, the user can set that personal account's username in `.Site.Params.opengraph.twitter.page`.

And as `twitter:site` is a required tag for Twitter Cards, we can check if `.Site.Params.opengraph.twitter.page` is set, and then conditionally render all the twitter tags.